### ISSUE TITLE
Do not include dishes as vernacular names in the Fishbase checklist

### ISSUE THREAD

**Issue by CecSve** [CatalogueOfLife/data#717]
**Describe the problem**:

This has also been reported through COL helpdesk and @DianRHR might be looking into the matter.

A user has noticed that checklists published by COL and EUNIS contain vernacular names that relate to dishes and not actual vernacular names: https://github.com/gbif/portal-feedback/issues/5492 and 
https://discourse.gbif.org/t/what-to-do-with-this-a-synonym-of-gadus-merlangus-linnaeus-1758-is-a-dish-apparently-and-not-a-fish/4673. 
 
It seems both checklists get the data from Fishbase ([COL example](https://www.catalogueoflife.org/data/taxon/3ZV5N)) and pull not only the vernacular names but also the ‘market name’. Would it be possible to update the checklist so that ‘market name’ is not included, please?

**Link to effected CoL webpages**:
https://www.catalogueoflife.org/data/dataset/1010

**Literature references**:
...

**Comment by yroskov** [comment]
Who said that "market name" is not a vernacular name? It is vernacular name. CoL just does not recognize subunits inside vernacular names.

**Comment by CecSve** [comment]
I tend to disagree on this matter. In many cases, the 'vernacular' name from the market name are local dishes, food products or derivatives of food products. The object is therefore not the species it should be a vernacular name for.

**Comment by DimEvil** [comment]
Not saying this could be possibly relevant information, but "drooggezouten gekruide zalm' is a derived product from "zalm" *Salmo salar*
And "Koudgerookte vlinders" a way of cooking "Wijting' :Dutch vernacular name for "Merlangius merlangus"
A market name is not the same as 'dwc:vernacularName' --> is a common name.

We are not going to make "chips" a local name for *Solanum tuberosum*

On Fishbase this links to market names: https://fishbase.se/ComNames/CommonNamesList.php?ID=29&GenusName=Merlangius&SpeciesName=merlangus&StockCode=39&market=y

And this links to common names: https://fishbase.se/ComNames/CommonNamesList.php?ID=29&GenusName=Merlangius&SpeciesName=merlangus&StockCode=39

**Comment by DimEvil** [comment]
Even I don'tknow what that means 'koud gerookte vlinders', :), as a Dutch speaking person.  And 'Vlinder(s)' is a common name for butterfly(ies), it is confusing.
Even a Google search  'koudgerookte vlinders'will only lead you to fishbase (and derivates) in this case. I can't see why 'Koudgerookte vlinders' could be a common name. This describes a way of preparing food.

Also on fishbase: Definition of Term
Koudgerookte vlinders
(English) Cold smoked block fillet of small haddock or whiting, sometimes dyed; available chilled or frozen. (See also: [cold smoked](https://fishbase.mnhn.fr/glossary/Glossary.php?q=cold+smoked&sc=is), [fillet](https://fishbase.mnhn.fr/glossary/Glossary.php?q=fillet&sc=is), [Frozen fish](https://fishbase.mnhn.fr/glossary/Glossary.php?q=Frozen+fish&sc=is))

Can be haddock or whiting, so not even one species

Another example could be 'Maatjes', for this one I could see the 'common name' as it is a joung haring (*Clupea harengus*) with no ofspring yet, but it is not a common name for *Clupea Harengus*

@yroskov So what would your definition of common names be ? 

*CoL believe that all common names are important for users. If a restaurant customer sees "Koudgerookte vlinders" on a menu, he would like to know what the experts at FishBase (and CoL) say about the species (or species in plural) that may be covered by this Dutch name. This is FishBase & CoL approach.*

I agree that all common names are important for users. But a common name should lead to a species and not to a recipe; 
'koudgerookte vlinders' is not a taxon no matter in what direction you would turn this. But probably and hopefully it is tasty.

Also I can't find Koudgerookte vlinders on CoL

On GBIF *Koudgerookte vlinders* is stated as the 'vernacular' or 'common' name on GBIF. *this is wrong* and should be 'Wijting'

Unfortunately we see this in more examples and should be adressed.

*CoL believe that all common names are important for users. If a restaurant customer sees "Koudgerookte vlinders" on a menu, he would like to know what the experts at FishBase (and CoL) say about the species (or species in plural) that may be covered by this Dutch name. This is FishBase & CoL approach.*

I can follow and agree on this statement, or reasoning, but the way it is done now, is not correct. +1 for dwc:recipe or dwc/foodDerivative :) so we can finally use GBIF when cooking
